tinygrad/runtime/ops_python.py

```diff
@@ -109,7 +113,8 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
           else:
             # TODO: add real cast
             if dtypes.is_int(dtype):
-              ul[i] = [int(x) for x in inp[0]]
+              if dtypes.is_unsigned(dtype): ul[i] = [int(256+x) if x < 0 else int(x) for x in inp[0]]
```

Comment:
WTF

Reply:
There is a 16 32.... u right, I'll do cast properly tmr.

Reply:
and add better overflow tests